pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.github.steveice10</groupId>
     <artifactId>mcauthlib</artifactId>
-    <version>1.4-SNAPSHOT</version>
+    <version>1.5-SNAPSHOT</version>
```

Comment:
I would recommend leaving it at 1.4-SNAPSHOT since there have been no changes yet for 1.4; this would be the first.